test/controllers/nonsipp/loansmadeoroutstanding/LoansListControllerSpec.scala

```diff
@@ -213,21 +213,30 @@ class LoansListControllerSpec extends ControllerBaseSpec {
 
     act.like(
       renderView(onPageLoadViewOnly, userAnswers = noLoansUserAnswers, optPreviousAnswers = Some(previousUserAnswers)) {
-        implicit app => implicit request =>
-          injected[ListView].apply(
-            form(new YesNoPageFormProvider()),
-            viewModel(
-              srn,
-              page,
-              mode = ViewOnlyMode,
-              List(),
-              schemeName,
-              viewOnlyViewModel = Some(viewOnlyViewModel.copy(viewOnlyUpdated = true))
+        implicit app =>
+          implicit request =>
+            injected[ListView].apply(
+              form(new YesNoPageFormProvider()),
+              viewModel(
+                srn,
+                page,
+                mode = ViewOnlyMode,
+                List(),
+                schemeName,
+                viewOnlyViewModel = Some(viewOnlyViewModel.copy(viewOnlyUpdated = true))
+              )
             )
-          )
       }.withName("OnPageLoadViewOnly renders ok with no loans")
     )
 
+    act.like(
```

Comment:
Could you please move this from "LoansListController in view only mode" to "LoansListController"? As it is not part of the view only mode.
Same for borrowing